pkg/describe/atmos.yaml

```diff
@@ -0,0 +1,267 @@
+# CLI config is loaded from the following locations (from lowest to highest priority):
```

Comment:
We have a lot of atmos.yaml files across this repo. Is it possible to consolidate them into a single one?